fem/ceed/util.hpp

```diff
@@ -17,6 +17,9 @@
 #include <ceed.h>
 #include <ceed/hash.h>
 #include <ceed/backend.h>  // for CeedOperatorField
+#if !CEED_VERSION_GE(0,10,0)
+#error MFEM requires a libCEED version superior to 0.10.0
```

Comment:
I'm not sure "superior to" is the right term to use here -- use either "greater than or equal to" or simply ">=".

Reply:
```suggestion
#error MFEM requires a libCEED version >= 0.10.0
```